CompositionRoot/ModuleRegistrar.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Immutable;
+using Autofac;
+
+namespace Messerli.CompositionRoot
+{
+    public class ModuleRegistrar : Module
+    {
+        private readonly ImmutableList<Register> _mockRegistrations;
```

Comment:
Why is this called `_mockRegistrations`?

Reply:
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/4/commits/43a7810c2cda586de25689d160d598303166a461